src/pages/SignUp/SignUp.module.css

```
@@ -0,0 +1,145 @@
+.container {
+  width: 100%;
+  min-height: 100vh;
+  padding: 24px;
+  background-image: url(../../assets/SignUpBackground.png);
+  background-size: cover;
+  background-position: center;
+
+  display: flex;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)